components/style/gecko/selector_parser.rs

```diff
@@ -92,6 +94,12 @@ impl ToCss for NonTSPseudoClass {
                         dest.write_str(&value)?;
                         return dest.write_char(')')
                     }, )*
+                    NonTSPseudoClass::Dir(ref dir) => {
+                        let value: String = dir.into();
+                        dest.write_str(concat!(":dir("))?;
+                        dest.write_str(&value)?;
```

Comment:
Also, I believe this is not correct, and should do escaping for the identifier... Though it's an existing bug.